frontend/src/components/Post/PostImage.js

```diff
@@ -0,0 +1,12 @@
+import '../../config';
+import React from "react";
+
+import "./PostImage.scss";
+
+export default function PostImage(pictureId) {
+	return (
+		<div className="image-wrapper">
+			<img src={`${global.config.BACKEND_URL}/picture/${pictureId.pictureId}.jpg`} alt={`${typeof(pictureId.pictureId)}}`} />
```

Comment:
What's the reasoning for that `alt`?

Reply:
Sorry, that's a remenant of me trying to figure out what I did wrong. I'll change it back to `alt={'${pictureId.pictureId}'}`